ui/app/routes/vault/cluster/secrets/backends.ts

```diff
@@ -6,10 +6,12 @@
 import Route from '@ember/routing/route';
 import { service } from '@ember/service';
 
-export default Route.extend({
-  store: service(),
+import type Store from '@ember-data/store';
+
+export default class ClientsRoute extends Route {
```

Comment:
This is my bad because I didn't make the change in my spike, but can you update the name here to match the route, so 
```suggestion
export default class SecretsBackends extends Route {
```